Blazorade.Bootstrap.Components.Showroom/ProgressBars.razor

```diff
@@ -0,0 +1,134 @@
+﻿@code {
+    Spacing headingMarginTop = Spacing.Five;
+}
+
+<Heading Level="HeadingLevel.H2">Progress/ProgressBar Component</Heading>
+<Paragraph>
+    The <Anchor Url="https://getbootstrap.com/docs/4.4/components/progress/" OpenInNewTab="true">Progress component</Anchor>
+    is used to display progress bars, including stacked bars, animated backgrounds, and text labels.
+</Paragraph>
+
+<DocsSection ComponentName="ProgressBar" />
+
+<Heading Id="basic-progress-bar" IsAnchor="true" Level="HeadingLevel.H3" MarginTop="@headingMarginTop">Basic Progress Bars</Heading>
+<Paragraph>
+    A basic progress bar example.
+</Paragraph>
+
+<Progress>
+    <ProgressBar MaxValue="100" MinValue="0" Value="0"></ProgressBar>
```

Comment:
```suggestion
    <ProgressBar MaxValue="100" MinValue="0" Value="10"></ProgressBar>
```

It's maybe better to have some value in order for the progress bar to show a bit.

Reply:
Done.